test-fixtures/update_application_services_version.py

```diff
@@ -0,0 +1,79 @@
+import os, json
+import requests
+import datetime
+from pprint import pprint
+from enum import Enum
+
+
+APPLICATION_SERVICES_TAGS_INFO = 'https://api.github.com/repos/mozilla/application-services/tags'
+CARTFILE = 'Cartfile'
+
+def available_tags():
+    try:
+        resp = requests.get(APPLICATION_SERVICES_TAGS_INFO, headers={'Authorization': 'Token {}'.format(secret.GITHUB_TOKEN)})
```

Comment:
I'm not sure about that way to use the github_token here... I mean is the way to use it in python script as a variable, but not sure about the `secret.` part to access it...

Reply:
I don't think you need a token here anyways on this get, or any auth, right?

```Python
import requests
data = requests.get("https://api.github.com/repos/mozilla/application-services/tags")
print(data.json())
```
Prints out the full request at /tags/

Reply:
@AaronMT We use the token so that we don't get the API limit errors if we query the github api too many times, I guess that for the use we are going to have here, it would not be necessary but it would not hurt either...

Reply:
There is this approach as in Android projects that I like, but not sure how/where to store the GITHUB_TOKEN @rpappalax do you know? if not we could ask people in one of the Android projects: 
```
import github

github_access_token = os.getenv("GITHUB_TOKEN")
GITHUB_REPO = "mozilla/application-services"

g = github.Github(github_access_token)
repo = g.get_repo(GITHUB_REPO)
latest_tag = repo.get_tags()[0].name
return (str(latest_tag))
```

Reply:
We could start with the approach without token, as implemented now after the changes, and improve it later to that second approach with token or wait until having that second option to start from there. 
Please let me know your thoughts. Thanks